### PR TITLE
Upgrade dependencies

### PR DESCRIPTION
**Pull request by k0nserv**
Upgrade most dependencies, crucially webpack and its related
dependencies. I did not touch React here.

**After this PR**
```
$ npm outdated

Package              Current  Wanted   Latest  Location
react                 16.4.2  16.4.2  16.12.0  backpack-docs
react-dom             16.4.2  16.4.2  16.12.0  backpack-docs
react-router-dom       4.3.1   4.3.1    5.1.2  backpack-docs
react-test-renderer   16.4.2  16.4.2  16.12.0  backpack-docs
```

+ [x] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack/blob/master/CONTRIBUTING.md)

Remember to include the following changes:
+ [x] [Links platform metadata](https://github.com/Skyscanner/backpack-docs/blob/master/docs/src/layouts/links.js)

_If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/master/CODE_REVIEW_GUIDELINES.md)_

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
     "flow": "flow",
     "flow-typed": "flow-typed",
     "start": "npm run docs",
-    "docs": "npm run submodules:pull && BPK_BUILT_AT=$( date -u +%s ) webpack-dev-server --open",
```
Comment (ojcurtis) [inline package.json]: Why are we no longer pulling the submodules as part of the starting process?

Comment (k0nserv) [reply package.json]: I'm restoring this I just tend to remove this line because I find it super annoying that it does this all the time

Comment (shaundon) [reply package.json]: Why not add a `docs:no-submodules-pull` command?

File: docs/src/constants/redirect-routes.js [hunk 2]
```diff
@@ -69,7 +69,6 @@ export default {
   '/components/native/navigation-bar': ROUTES.NAVIGATION_BAR,
   '/components/native/navigation-stack': ROUTES.NAVIGATION_STACK,
   '/components/native/nudger': ROUTES.NUDGER,
-  '/components/native/pagination-dots': ROUTES.PAGINATION_DOT,
```
Comment (shaundon) [inline docs/src/constants/redirect-routes.js]: Why's this been removed?

Comment (k0nserv) [reply docs/src/constants/redirect-routes.js]: `ROUTES.PAGINATION_DOT` does not exist. https://backpack.github.io/components/native/pagination-dots/ is just a broken page

File: package.json (hunk 1 above)
Comment (k0nserv) [reply package.json]: Because I'm lazy

File: docs/src/constants/redirect-routes.js (hunk 2 above)
Comment (shaundon) [reply docs/src/constants/redirect-routes.js]: Dayum

File: babel.config.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2019 Skyscanner Ltd
```
Comment (shaundon) [inline babel.config.js]:
```suggestion
 * Copyright 2016-2020 Skyscanner Ltd
```

**Pull request merged**